src/App.js

```diff
@@ -4,9 +4,23 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    movies: moviesFromServer,
+  };
+
+  filterMovies = (event) => {
+    const { value } = event.target;
+
+    this.setState({
+      movies: moviesFromServer.filter(movie => (
+        movie.title.toLowerCase().includes(value.toLocaleLowerCase())
+      )),
```

Comment:
```suggestion
      movies: moviesFromServer.filter(movie => (
        movie.title.toLowerCase().includes(value.toLocaleLowerCase())
      )),
```
![list](https://user-images.githubusercontent.com/77117136/112202590-3c154f80-8c1a-11eb-8361-28092446c74d.png)